webapp/pom.xml

```diff
@@ -240,7 +241,6 @@
             <groupId>com.box.l10n.mojito</groupId>
             <artifactId>mojito-restclient</artifactId>
             <version>0.111-SNAPSHOT</version>
-            <scope>test</scope>
```

Comment:
Removing the test scope to use `ThirdPartySync` in `ThirdPartyWS` and `ThirdPartyService`.